pyproject.toml

```diff
@@ -5,11 +5,9 @@ backend-path = ["."]
 
 [tool.black]
 skip-string-normalization = true
+extend_exclude = "_vendor"
```

Comment:
Should we add all the 3rd party and autogenerated code here as well? (i.e. the folders: `setuptool/_distutils` and `setuptools/config/_validate_pyproject`?)

I have a doubt, will `ruff` also follow this config?

Reply:
Maybe? My instinct is not to add it until needed. I'm pretty sure `_distutils` follows black now... and since `_validate_pyproject` isn't complaining, let's let it get the black treatment until it becomes untenable to do so (upstream introduces incompatible syntax).

I would not expect `ruff` to follow this config, but it seems to be satisfied by the default config, so I'm inclined to stick with it.